mantidproject/mantid
# Add algorithm EnggVanadiumCorrections

Fixes #13184.
Fixes #13139.

This adds an algorithm with mostly the same Vanadium related code that was previously added in scripts/EnggUtils.py. There are additional changes to support the two additional inputs (VanIntegration and VanCurves) in the other Engg algorithms that use vanadium corrections.

**To test**:
- it comes with unit and doc tests which should pass
- review of code and algorithm documentation

This also removes from the docs-test UsageData directory two files that are too big to be in the doctest data directory (addressing #13139):
- long ceria run ENGINX00193749.nxs
- Vanadium run ENGINX00236516.nxs
